tests/test_meshtastic_utils.py

```
@@ -4644,13 +4636,8 @@ def test_disconnect_ble_by_address_is_connected_bool_true(
         """Test _disconnect_ble_by_address when is_connected is a bool (True)."""
         from mmrelay.meshtastic_utils import _disconnect_ble_by_address
 
-        async def _noop(*_args, **_kwargs):
-            """
-            Asynchronous no-op that ignores all positional and keyword arguments.
-
-            Returns:
-                None: Always returns None.
-            """
+        def _noop(*_args, **_kwargs):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_noop`

Add return type annotation: `None`

(ANN202)